Source/Core/Mixin.lua

```diff
@@ -60,8 +62,10 @@ end
 
 function AuctionatorAHFrameMixin:ToggleShoppingLists()
   if AuctionatorShoppingLists:IsVisible() then
+    Auctionator.Config.Set(Auctionator.Config.Option.SHOW_LISTS, false);
```

Comment:
I don't think toggling the visibility should change the configuration setting? I read the config as "I do (or don't) want the Shopping List window to show up by default when the AH is opened". I may just want to hide or show the window temporarily while in the AH without affecting the behavior I want on AH open.

Reply:
For me using the AH it feels irritating that it keeps showing even after I've closed it. Just preference - I like things to stay the way I left them